### ISSUE TITLE
Any way to order it 'desc' by date timestamp?

### ISSUE THREAD

**Issue by hellseyfer** [AngularFirebase/145-infinite-virtual-scroll-cdk-angular#1]
When I try to do this. it gives me an error. I think because some docs are not loaded yet

**Comment by bryananthonylewis** [comment]
I don't think you can use startAfter if you use 'desc'. Not 100% sure but that's my guess. I'm having issues getting data though. this.infinite doesn't have an array in it. 🤷🏼‍♂️
